tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -66,6 +66,9 @@ describe('util/extension-runners/chromium', async () => {
       '--password-store=basic',
       '--use-mock-keychain',
       '--force-fieldtrials=*BackgroundTracing/default/',
+      '--disable-hang-monitor',
+      '--disable-prompt-on-repost',
+      '--disable-domain-reliability',
```

Comment:
These changes were required after upgrading chrome-launcher, similar to https://github.com/mozilla/web-ext/pull/2825#discussion_r1268584304